content/docs/command-reference/root.md

```diff
@@ -10,11 +10,12 @@ usage: dvc root [-h] [-q | -v]
 
 ## Description
 
-This command returns the path to the root directory of the <abbr>DVC
-project</abbr>, relative to the current working directory. It can be used to
-build a path to a dependency, script, or <abbr>data artifact</abbr>, for
-example. Useful when working in a subdirectory of the project, and needing to
-refer to a file in another directory.
+Returns the path to the root directory of the <abbr>DVC project</abbr>, relative
+to the current working directory. `dvc root` is useful for referencing a file
+in another directory when working in a subdirectory of the project.
+
+Use this command to build reusable paths to dependencies, scripts, or
+<abbr>data artifacts</abbr>.
```

Comment:
What is a "reusable path"? Still needs more explanation on the command motivation or main use case(s) as discussed in #404. It's OK if the paragraph grows (in fact I was expecting so, thus recommended separating this into its own p).

Reply:
OK, working on expanding this to include scripting motivation.

Reply:
Expanded this with another sentence. The POSIX use case seems platform-specific so I left it out.